SiraUtil/Tools/FPFC/FPFCToggle.cs

```diff
@@ -186,8 +186,6 @@ private void DisableFPFC()
         public void Dispose()
         {
             _fpfcSettings.Changed -= FPFCSettings_Changed;
-
-            InitializeXRLoader();
```

Comment:
I initially misunderstood when `Dispose` runs for this class (I didn't realize this happened for each instance of the FPFC, I thought it was one `FPFCToggle` instance for the whole game) so calling `InitializeXRLoader` here doesn't really make sense and just slows things down.